specs/mediawiki/v1/content.yaml

```diff
@@ -840,6 +903,9 @@ paths:
             $ref: '#/definitions/problem'
       x-backend-request:
         uri: /{domain}/sys/parsoid/transform/wikitext/to/html{/title}{/revision}
+        body:
+          wikitext: '{wikitext}'
+          bodyOnly: '{bodyOnly}'
```

Comment:
But, the RESTBase interface translates that, so that this should work as-is: https://github.com/wikimedia/mediawiki/blob/b79196778ed7c4f49ae09bee9ba16f861a6e3760/includes/libs/virtualrest/RestbaseVirtualRESTService.php#L153